stable/grafana/templates/secret.yaml

```diff
@@ -15,3 +15,6 @@ data:
   {{- else }}
   admin-password: {{ randAlphaNum 40 | b64enc | quote }}
   {{- end }}
+  {{- if index .Values "ldap.toml" }}
+  ldap.toml: {{ include "ldap.toml" . | b64enc | quote }}
```

Comment:
This could have been simplified with `toToml`, but I encountered https://github.com/kubernetes/helm/issues/2474. But Grafana was unable to parse floats or strings to an int value